content/blog/20230829_fluent-package-scheduled-lifecycle.md

```diff
@@ -0,0 +1,113 @@
+# Scheduled support lifecycle announcement about Fluent Package
+
+Hi users!
+
+We have released `fluent-package` [v5.0.0](https://github.com/fluent/fluent-package-builder/releases/tag/v5.0.0) (Release Candidate) and 
+the following [v5.0.1](https://github.com/fluent/fluent-package-builder/releases/tag/v5.0.1) (General Availability).
+fluent-package is formerly known as `td-agent` and a stable distribution package of Fluentd.
+
+In this blog article, we explains why the concept of channels is introduced.
+
+### Why is LTS (Long Term Support) channel introduced?
+
+In the previous versions, `td-agent` was shipped as quarterly releases in most cases.
+It sometimes contains fluentd minor upgrade occasionally, it means that not only bug or security fixes,
+but also contains new features. As a result, there was a case that the stability was affected unexpectedly.
+
+In point view of adopting fluentd (and `td-agent`) for enterprise services, there was a requirement for the distribution which contains security or bug fixes only
+and scheduled lifecycle of it.
+
+The LTS release is the answer to support them.
+
+### Channels for updates
+
+There are two channels of support lifecycle.
+
+* Standard
+* LTS (Long Term Support)
+
+Here is the difference of these channels.
+
+* Standard (5.x series)
+  * Recommended for early adopters (eager to try new features of Fluentd)
+  * Rapid release development (planned semi-annually releases, but it may vary.)
+  * Fluentd will be upgraded occasionally
+      * Minor upgrade will be applied. e.g. 1.16 => 1.17 and so on.
+      * e.g. v5.0.1 (Fluentd v1.16.2), v5.0.2 (Fluentd v1.16.3), ... v5.1.0 (Fluentd v1.17.0)
+  * The only latest version will be supported (bug fix will not be backported into older version)
+* LTS (Long Term Support, 5.0.x series)
+  * Recommended for enterprise services
+  * Security and bug fix release only
+  * Fluentd will be upgraded (teeny upgrade only. it will stick to 1.16.x series because it is aimed to fix security or bug issues.)
```

Comment:
I feel that the content in the brackets would make it harder to understand.
Wouldn't it be easier to understand if it were expressed simply in one sentence?

```suggestion
  * Fluentd will be upgraded only in the teeny version and will stick to 1.16.x series.
```